src/APBFramelessWindow/APBPushButton.cpp

```diff
@@ -69,15 +68,13 @@ bool APBPushButton::event(QEvent *event) {
 				if(this->timer != nullptr) {
 					this->stopTimer(&palette);
 				}
-
```

Comment:
Clean code principle: `Use white space to associate related things and disassociate weakly related.`